scripts/make_urbansound8k_index.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import hashlib
+import json
+import os
+import glob
+
+INDEX_PATH = "../soundata/datasets/indexes/urbansound8k_index.json"
+
+
+def md5(file_path):
+    """Get md5 hash of a file.
+
+    Parameters
+    ----------
+    file_path: str
+        File path.
+
+    Returns
+    -------
+    md5_hash: str
+        md5 hash of data in file_path
+    """
+    hash_md5 = hashlib.md5()
+    with open(file_path, "rb") as fhandle:
+        for chunk in iter(lambda: fhandle.read(4096), b""):
+            hash_md5.update(chunk)
+    return hash_md5.hexdigest()
```

Comment:
nit: I think we should import this from `validate` instead of adding this code to each `make_index_dataset.py` we write

Reply:
agreed, fixed